orc8r/lib/go/build_info/build_info.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// package build_info provides API to get information on the binary build/commit version
```

Comment:
can we make a note those variables will be set at runtime using `ldflags`

This way next persons who reads it will have to just search for ldflags to find it where is set